api/server.spec.js

```diff
@@ -1,40 +1,57 @@
 const mongoose = require('mongoose');
+const request = require( 'supertest' );
+//const server = require( './api/server' );
+const Game = require( '../games/Game' );
 
-const Game = require('./games/Game');
-
-describe('The API Server', () => {
+describe('Game', () => {
   beforeAll(() => {
     return mongoose
       .connect('mongodb://localhost/test')
       .then(() => console.log('\n=== connected to TEST DB ==='))
       .catch(err => {
         console.log('error connecting to TEST database, is MongoDB running?');
       });
-  });
+  } );
+  
+  
 
   afterAll(() => {
     return mongoose
       .disconnect()
       .then(() => console.log('\n=== disconnected from TEST DB ==='));
   });
 
-  let gameId;
+  let gameId, game;
   // // hint - these wont be constants because you'll need to override them.
 
-  beforeEach(() => {
+  beforeEach(async() => {
     //   // write a beforeEach hook that will populate your test DB with data
     //   // each time this hook runs, you should save a document to your db
-    //   // by saving the document you'll be able to use it in each of your `it` blocks
+    //   // by saving the document you'll be able to use it in each of your `it` 
+    game = await Game.create( { title: 'playgame', genre: 'play' } );
+    gameId = game._id;
+
   });
 
   afterEach(() => {
     //   // clear the games collection.
+    return Game.remove();
   });
-
+  
   it('runs the tests', () => {});
 
+
   // test the POST here
+  it( 'POST', async () => {
+
+    const response = await request( server ).post( './games/Game' ).send(game);
```

Comment:
consider posting to '/games/game' or something following that pattern? the period and capital letters are very unusual in a post endpoint.